### ISSUE TITLE
using a closure as argument causes odd errors

### ISSUE THREAD

**Issue by joehoyle** [WordPress-Coding-Standards/WordPress-Coding-Standards#525]
Appears to be a dupe of https://github.com/squizlabs/PHP_CodeSniffer/issues/318 but still happening with me (on latest dev). Running against the following:

```PHP
<?php
hm_add_rewrite_rule( array(
	'regex'            => '^emails/change-city/?$',
	'permission'       => 'logged_in_only',
	'request_callback' => function() {
		$user = User::current_user();
		return;
	},
));
```

I get:

```
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 7 | ERROR | [x] Multi-line function call not indented correctly;
   |       |     expected 4 spaces but found 8
 8 | ERROR | [x] Multi-line function call not indented correctly;
   |       |     expected 4 spaces but found 8
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

Not sure exactly what's going on here. If I don't have the `return;` line, it doesn't fail, but once I add any more lines I get these errors.

**Comment by westonruter** [comment]
I am seeing the same issue with `PEAR.Functions.FunctionCallSignature.Indent`. In 2.3.4 it works as expected, but in the current 2.5.1 it is broken.

I ran git bisect to pinpoint the problem and it identified that the issue with  was introduced in https://github.com/squizlabs/PHP_CodeSniffer/commit/3f212a0e4278c4a5dcb2f5ed3c46c82fa507979f as part of #698.

**Comment by joehoyle** [comment]
Downgraded to 2.3.4 and confirmed that as functional. Thanks for the workaround @westonruter!

**Comment by westonruter** [comment]
@joehoyle looks like the issue is now fixed in PHPCS! https://github.com/squizlabs/PHP_CodeSniffer/issues/698#issuecomment-180609565

**Comment by joehoyle** [comment]
Great stuff! I'll close this ticket out now, thanks for coordinating.
